misago/users/tests/test_anonymizeoldips.py

```diff
@@ -1,43 +1,45 @@
 import datetime
-import pytz
 
 from django.contrib.auth import get_user_model
 from django.core.management import call_command
-from django.test import TestCase
-from django.utils.six import StringIO
 from django.utils import timezone
+from django.utils.six import StringIO
+from django.test import TestCase, override_settings
 
-from misago.conf import settings
-from misago.users.management.commands import anonymizeoldips
 from misago.core.utils import ANONYMOUS_IP
+from misago.users.management.commands import anonymizeoldips
 
 
-# - command is not anonymizing user's IP if anonymization is disabled
-# - command is anonymizing user's IP if its old
 
 UserModel = get_user_model()
 
 
 class AnonymizeOldIpsTests(TestCase):
-    def setUp(self):
-        self.user = UserModel.objects.create_user('Bob', 'bob@bob.com', 'pass123')
-        self.user_with_joined_on = UserModel.objects.create_user('Bob1', 'bob1@bob.com', 'pass1231')
-        self.user_with_joined_on.joined_on = datetime.datetime(2013, 11, 20, 20, 8, 7, 127325, tzinfo=pytz.UTC)
-        self.user_with_joined_on.save()
-
     def test_anonymizeoldips_new_user(self):
-        """ command is not anonymizing user's IP if its new """
-
+        """command is not anonymizing user's IP if its new"""
+        user = UserModel.objects.create_user('Bob', 'bob@bob.com')
         call_command(anonymizeoldips.Command())
-        user_joined_from_ip = UserModel.objects.get(pk=self.user.pk).joined_from_ip
-
-        self.assertFalse(user_joined_from_ip == ANONYMOUS_IP, msg=None)
+        user_joined_from_ip = UserModel.objects.get(pk=user.pk).joined_from_ip
+        
+        self.assertNotEqual(user_joined_from_ip, ANONYMOUS_IP)
     
     def test_anonymizeoldips_old_user(self):
-        """ command is anonymizing user's IP if its old """
+        """command is anonymizing user's IP if its old"""
+        current_datetime = timezone.now()
+        datetime_50_days_past = current_datetime - datetime.timedelta(days=51)
+        user = UserModel.objects.create_user('Bob', 'bob@bob.com')
+        user.joined_on = datetime_50_days_past
+        user.save()
         call_command(anonymizeoldips.Command())
+        user_joined_from_ip = UserModel.objects.get(pk=user.pk).joined_from_ip
+
+        self.assertEqual(user_joined_from_ip, ANONYMOUS_IP)
 
-        user_with_joined_on = UserModel.objects.get(pk=self.user_with_joined_on.pk)
-        timediff = datetime.datetime.now(timezone.utc) - user_with_joined_on.joined_on
+    @override_settings(MISAGO_IP_STORE_TIME = None)
+    def test_not_anonymizing_user_ip(self):
```

Comment:
This test should check if user with old account has their `joined_from_ip` unhanged after `anonymizeoldips` is ran with `MISAGO_IP_STORE_TIME=None`, but don't remove the `self.assertEqual(command_output, "IP anonymization is disabled.")`, this is also good for including in this test.